src/app/static/src/app/components/plots/bubble/utils.ts

```diff
@@ -38,14 +38,21 @@ export const getFeatureIndicators = function (data: any[],
             if (indicatorMeta.indicator == colorIndicator.indicator) {
                 result[areaId].value = value
                 result[areaId].color = getColor(value, indicatorMeta, colourRange)
-                result[areaId].lower_value = (values['lower'] || values['lower'] === 0) ? values['lower'] : ""
-                result[areaId].upper_value = values['upper']
+                if(values['lower'])
```

Comment:
This comment is still unaddressed, and it still seems me that 0s will remain undefined, which we don't want as 0s are valid vald values. Could you please explain your thinking here, and also add a test for tooltips where lower or upperbound is 0? 

Why not just pass these values through without the check, and then do the `value ||  value === 0` checks in the tooltip code?

Sorry if this has got really confusing, let's chat about it if not clear. 

>So this won't include 0s? lower_value or upper_value will remain undefined if data value is 0? I thought we did want to pass those through?